network-api/networkapi/campaign/views.py

```diff
@@ -194,6 +203,13 @@ def petition_submission(request, petition):
         }
     })
 
+    if settings.MOFO_NEWSLETTER_SUBSCRIBE_METHOD == 'BASKET':
```

Comment:
```suggestion
    if settings.MOFO_NEWSLETTER_SUBSCRIBE_METHOD == 'BASKET' \
        and request.data['newsletterSignup'] is True:
```
I believe this is the correct way to combine the two if statements into one statement while following the [PEP8 79 character line limit](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). I believe linter should automatically do the right thing.